Sources/ComposableArchitecture/Effect.swift

```diff
@@ -158,13 +158,13 @@ public struct Effect<Output, Failure: Error>: Publisher {
   ///       }
   ///     }
   ///
-  /// - Parameter run: A closure that accepts a `Subscriber` value and returns a cancellable. When
-  ///   the `Effect` is completed, the cancellable will be used to clean up any
-  ///   resources created when the effect was started.
+  /// - Parameter work: A closure that accepts a `Subscriber` value and returns a cancellable. When
+  ///   the `Effect` is completed, the cancellable will be used to clean up any resources created
+  ///   when the effect was started.
   public static func async(
-    _ run: @escaping (Effect.Subscriber<Output, Failure>) -> Cancellable
+    _ work: @escaping (Effect.Subscriber<Output, Failure>) -> Cancellable
```

Comment:
We use `run` some places and `work` other places. In fact I wonder if this entire thing could be renamed from `async` to `run` 🤔

`Effect.run { subscriber in ... }` has a nice ring to it...

Reply:
Dang, yeah. That’s pretty nice

Reply:
Should we make the change in this PR? Separate? Deprecate with `renamed:`?

Reply:
I'll merge as is. We can think about this a bit more.